Sources/Commands/SwiftTestCommand.swift

```diff
@@ -188,6 +188,15 @@ struct TestCommandOptions: ParsableArguments {
             help: "Path where the xUnit xml file should be generated.")
     var xUnitOutput: AbsolutePath?
 
+    @Flag(
+        name: .customLong("xunit-message-failure"),
```

Comment:
Experimental flags are typically prefixed with `experimental` to make it clear at the callsite the flag is prone to change. This would become `experimental-xunit-message-failure`

Reply:
isnt' the fact that it's a hidden argument indicative it's experimental?  if we prefix with `experimental`, then why does the flag need to be hidden?

Reply:
Hidden flags are not always experimental. Prefixing experimental flags with `--experimental` indicates they're liable to be changed or removed. It also has the added benefit that its clear to consumers that a command is using unstable API. There is no way to tell that at a glance `swift test --xunit-message-failure` is using an API that will change in the future.

Reply:
I'm ok with prefixing `experimental`, but to make it discoverable, I would favour not hiding the argument.

Reply:
There are three levels of flag visibility, all (default), `.hidden`, and `.private`. `.hidden` shows in the extended help display, `.private` doesn't show the flag anywhere.

Typically experimental flags are `.hidden` so we don't encourage adoption, but they are documented in the extended help. `.private` flags are usually "internal only" and not intended to be consumed by anyone outside of the project. I think there is an argument here that this flag should be `.private` and not `.hidden` if the goal is just to add a basic failure message to SPMs xunit reports. 

The `--experimental` prefix is different than flag visibilty, it denotes a flag that may be changed or removed. So in combination both `.private` and experimental means this is "a flag that is subject to change for the project's internal use".

Reply:
I prefixed the flag with `--experimental` and kept the visibility `.hidden` as it's nice to have it discoverable.